transfers/errors.py

```diff
@@ -3,13 +3,15 @@
 ERR_INVALID_RESPONSE = 1
 ERR_PARSE_JSON = 2
 ERR_SERVER_CONN = 3
+ERR_INVALID_URL = 4
 ERR_CLIENT_UNKNOWN = -1
 
 error_codes = {
     ERR_INVALID_RESPONSE: "Invalid response from server, check amclient log",
     ERR_PARSE_JSON: "Could not parse JSON resposne, check amclient log",
     ERR_SERVER_CONN: "Error connecting to the server, check amclient log",
-    ERR_CLIENT_UNKNOWN: "Unknown return from amclient, check logs."}
+    ERR_CLIENT_UNKNOWN: "Unknown return from amclient, check logs",
+    ERR_INVALID_URL: "Invalid URL passed to AM Client"}
```

Comment:
You may want to move the closing bracket to its own line and include the comma in the last item. It's a convention in Python that reduces risk and the size of diffs when the structure is edited.

Reply:
```python
error_codes = {
    ERR_INVALID_RESPONSE: "Invalid response from server, check amclient log",
    ERR_PARSE_JSON: "Could not parse JSON resposne, check amclient log",
    ERR_SERVER_CONN: "Error connecting to the server, check amclient log",
    ERR_CLIENT_UNKNOWN: "Unknown return from amclient, check logs",
    ERR_INVALID_URL: "Invalid URL passed to AM Client",
}
```

Reply:
:+1: